lib/backend/firestore/firestorebk_test.go

```diff
@@ -87,7 +102,17 @@ func ensureTestsEnabled(t *testing.T) {
 }
 
 func ensureEmulatorRunning(t *testing.T, cfg map[string]interface{}) {
-	con, err := net.Dial("tcp", cfg["endpoint"].(string))
+	v, ok := cfg["endpoint"]
+	if !ok {
+		return
+	}
+
+	endpoint, ok := v.(string)
+	if !ok || endpoint == "" {
+		return
```

Comment:
```suggestion
		t.Skip("The Firestore emulator is not configured")
```

or something along those lines, rather than passing the test without testing anything

Reply:
oh, nevermind then!